_layouts/cask.html

```diff
@@ -83,6 +83,26 @@
     </tr>
 </table>
 {%- endif %}
+{%- if c.variations.size > 0 -%}
+<p>Variations:</p>
+<table class="full-width">
+    <thead>
+        <th>OS</td>
+        <th>Version</td>
+    </thead>
+    {%- for variation in c.variations -%}
+    <tr>
+        <td>{{ variation[0] }}</td>
+        <td>
+            <a href="{{ variation[1].url | default: c.url }}"
+               title="Download for {{ c.token }} on {{ variation[0] }}">
+                {{ variation[1].version | default: c.version }}
+            </a>
+        </td>
```

Comment:
As mentioned by @Bo98 please use some of the code in https://github.com/Homebrew/formulae.brew.sh/blob/bee20d346b5cb2b3d5bd95254033bdc50581fcff/_layouts/formula.html#L60-L89 to:

- have separate Apple Silicon and Intel groups
- capitalise and use `remove_first: "arm64_" | replace: "_", "&nbsp;"`